src/containers/ConceptPage/components/ConceptForm.jsx

```diff
@@ -344,13 +346,14 @@ class ConceptForm extends Component {
                   errors={errors}
                   getEntity={this.getConcept}
                   entityStatus={concept.status}
-                  createMessage={this.props.createMessage}
+                  createMessage={createMessage}
                   showSimpleFooter={!concept.id}
                   setFieldValue={setFieldValue}
                   onSaveClick={() => {
                     this.handleSubmit(formikProps);
                   }}
-                  entityType={'Concept'}
+                  getStateStatuses={fetchStateStatuses}
+                  isArticle={false}
```

Comment:
Denne trur eg du kan droppe. Meiner manglende prop tolkes som false.

Reply:
Du har rett, bare tenker at det er mer oversiktig å se hva slags props blir sendt og ikke, lettere å debugge og blir mer lesbart.

Reply:
Når jeg skriver kode spesifiserer jeg alltid hva slags props som sendes inn istedenfor å bruke `...rest` for da er koden mer lesbart og man slipper å console.logge() hva `rest` inneholder. Så det med isArticle={false} gjør jeg av samme prinsipp

Reply:
Klarer ikke se den sammenligninga her @N4G1 :joy:
Jeg er enig med deg i at å spreade props er en uting som gjør koden vanskeligere å lese (med mindre det er nødvendig), men jeg syns `isArticle` er penere enn `isArticle={true}`.

I dette tilfelle så vil vel aldri propen inneholde noe annet enn `true` eller `undefined`?
Med mindre den finnes i en `...rest` spread da, men da vil den jo overskrives uansett?

Reply:
Mente det med `...rest` ja, med at hvis du har ...rest så vet ikke du om propp'en er false eller om den ble sendt inn i ...rest. Så for meg gir det mening å ha ting eksplisitt som `isArticle={false}`. Enig i at `isArticle={true}` er overkill og kan gjøres til `isArticle`. Og i dette tilfelle hvor jeg ikke spredder noen props så kan jeg forsåvidt fjerne `isArticle` og da vil den bli implisitt false, men igjen, litt sånn implisitt vs eksplisitt kamp.

Reply:
Da kan man ikke ha props med default verdier da, og alle props må være required? 
I tilfelle noen vil bruke spreading?

Reply:
hmm, godt poeng, har ikke tenkt over default verdier på props. Jeg blir bare så sliten av å se `...rest` for da vet jeg at jeg må console.logge for å finne hva den inneholder, så prøver å fjerne mest mulig av de når jeg kan, men ser at kan heller ikke være for nazi på det.

Reply:
Du må jo ikkje logge for å finne verdiene. Om du debugger eller bruker react-tools ser du jo kva for verdier som sendes inn til komponenten. Enig at det er vanskeligere å lese rest-spreads men vi klarer ikkje unngå det.